File: customer-db/src/main/java/org/javatots/example/customerdb/models/Voicable.java
package org.javatots.example.customerdb.models;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Voicable {
    private String type;
    private int number;

    @Override
    public String toString() {
        return "Voicable{" +
                "type='" + this.type + '\'' +
                ", number=" + this.number +
                '}';
    }
}

File: asTypescript/packages/custdb/src/records/Voicable.ts
// Corresponding javatots-test package: org.javatots.example.customerdb.models
export class Voicable {

   private type: string;

   private number: number;

  public toString(): string {
    return "Voicable{" + "type='" + this.type + '\'' + ", number=" + this.number + '}';
  }

  public constructor() {
  }

  public constructor(type: string, number: number) {
    this.type = type;
    this.number = number;
  }

  public setType(type: string): void {
    this.type = type;
  }

  public getType(): string {
    return this.type;
  }

  public setNumber(number: number): void {
    this.number = number;
  }

  public getNumber(): number {
    return this.number;
  }
}
